### PR TITLE
Refactor/contributing

### PR DESCRIPTION
**Pull request by dieppa**
internal

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,28 +1,35 @@
- # Contributing to Mongock project
+# Contributing to Mongock project
 
 :tada: First off, thanks for taking the time to contribute! :tada:
 
  The following is a set of guidelines for contributing to Mongock project which is hosted on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel
```
Comment (dhivesh) [inline CONTRIBUTING.md]: *judgement

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
  free to propose changes to this document in a pull request.
 
 
-## Getting started
+## Getting started :rocket:
 
-> :boom::rocket: Visit our page of [requested contribution](https://github.com/mongock/mongock/labels/contribution-requested) to find out the best candidates and which of them are rewarded!!
+> :boom: Visit our page of [requested contribution](https://github.com/mongock/mongock/labels/contribution-requested) to find out the best candidates and which of them are rewarded!!
```
Comment (dhivesh) [inline CONTRIBUTING.md]: Update with:
> :boom: Visit our page of [requested contribution](https://github.com/mongock/mongock/labels/contribution-requested) to find out issues and which of them are rewarded!!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,28 +1,35 @@
...
-
-
-## Issues
+## Issues :bug:
 For raising a new issue, please visit this [link](https://github.com/mongock/mongock/issues/new?assignees=&labels=&template=bug_report.md&title=), fills the template and submit it. 
 You will hear from us shortly.
 
-## Discussions
-For general **Q&A**s, **Ideas**, **comment on any release** or anything you want to raise, but it's not an issue, please visit the [discussion page](https://github.com/mongock/mongock/discussions)
+**Contributor raising a valid issue(or solving it) will get listed in our [contributor list](https://github.com/mongock/mongock/blob/master/CONTRIBUTORS.md).**
```
Comment (dhivesh) [inline CONTRIBUTING.md]: Can we perhaps manage the contributors list as:
- Collaborators (one-off contributions)
- Contributors (code maintainers) 
We can also have a separate definition - just an idea :)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,28 +1,35 @@
...
+
+## Discussions :loud_sound:
+For general **Q&A**s, **ideas**, comment on any **release** or anything you want to raise, but it's not an issue, please visit the [discussion page](https://github.com/mongock/mongock/discussions).
+
+
+**Contributor starting a valid discussion(or participates in it) will get listed in our [contributor list](https://github.com/mongock/mongock/blob/master/CONTRIBUTORS.md).**
```
Comment (dhivesh) [inline CONTRIBUTING.md]: Agree, we could and should mention anyone who is helping. But would we want to differenciate in what someone is helping on given the potential effort?

File: CONTRIBUTING.md (hunk 3 above)
Comment (dieppa) [reply CONTRIBUTING.md]: I agree. Somewhere ther I mention "external collaboration" to make that difference. We may need to work in separate those two concepts.

File: CONTRIBUTING.md (hunk 4 above)
Comment (dieppa) [reply CONTRIBUTING.md]: Take a look to the `code contribution` section, it says , they will be listed with special mention

Is that what you mean?

**Pull request merged**